channel.go

```diff
@@ -894,11 +1118,67 @@ closed with an error.
 
 */
 func (ch *Channel) QueueBind(name, key, exchange string, noWait bool, args Table) error {
+	return ch.QueueBindContext(
+		context.Background(),
+		name,
+		key,
+		exchange,
+		noWait,
+		args,
+	)
+}
+
+/*
+QueueBindContext binds an exchange to a queue so that publishings to the
+exchange will be routed to the queue when the publishing routing key matches the
```

Comment:
Some exchanges do not use routing keys or the idea of "matching" is quite a stretch. Should we link to https://www.rabbitmq.com/getstarted.html instead?